index.md

```diff
@@ -59,4 +64,3 @@ Join our announcements email list!
   <button class="btn btn--primary" style="font-size: 1em">Subscribe</button>
 </div>
 {: .text-center}
-
```

Comment:
I'd generally want to see unrelated whitespace changes in a separate commit or even PR, but that's just a note for the future. I will be approving this.